draft-ietf-rats-corim.md

```diff
@@ -821,7 +832,7 @@ An instance carries a unique identifier that is reliably bound to a Target Envir
 that is an instance of the Attester.
 
 The types defined for an instance identifier are CBOR tagged expressions of
-UEID, UUID, or cryptographic key identifier.
+UEID, UUID, variable-length opaque byte string, or cryptographic key identifier.
```

Comment:
Please either add a reference to tagged bytes or describe profile specific behaviour dependency here!